bluer/examples/mesh_provisioner.rs

```diff
@@ -0,0 +1,168 @@
+#![feature(generic_associated_types)]
+//! Attach and send/receive BT Mesh messages
```

Comment:
Would this be useful as a standalone tool?
Then consider moving it to bluer-tools and properly documenting the CLI arguments.

Reply:
No, this is just an example for now (as it is tied to the predefined model) ... The proper tool needs more work